### ISSUE TITLE
combineAll image does not have alt text.

### ISSUE THREAD

**Issue by chrisdemars** [ReactiveX/rxjs#3179]
The combineAll image in the operators section does not have an alt attribuite associated with it.

**Comment by martinsik** [comment]
I think you can close this one. It should be fixed in #3846.

**Comment by JWO719** [comment]
closed by #3846
